examples/plot_spm_multimodal_faces.py

```diff
@@ -70,24 +70,20 @@
     design_matrices.append(design_matrix)
 
 # specify contrasts
-_, matrix, names = check_design_matrix(design_matrix)
-contrasts = {}
-n_columns = len(names)
-contrast_matrix = np.eye(n_columns)
-for i in range(2):
-    contrasts[names[2 * i]] = contrast_matrix[2 * i]
-
+contrast_matrix = np.eye(design_matrix.shape[1])
+contrasts = dict([(column, contrast_matrix[i])
+                  for i, column in enumerate(design_matrix.columns)])
 # more interesting contrasts
-contrasts['faces-scrambled'] = contrasts['faces'] - contrasts['scrambled']
-contrasts['scrambled-faces'] = -contrasts['faces-scrambled']
-contrasts['effects_of_interest'] = np.vstack((contrasts['faces'],
-                                              contrasts['scrambled']))
+interesting_contrasts = {
```

Comment:
The point is to avoid clutter by showing figures on 25 contrasts...
What we can do instead is to simplify the code. Please comment on my commit.